tap_eloqua/client.py

```diff
@@ -35,12 +35,18 @@ def __enter__(self):
     def __exit__(self, type, value, traceback):
         self.__session.close()
 
-    @backoff.on_exception(backoff.expo,
-                          Server5xxError,
-                          max_tries=5,
-                          factor=2)
+    @backoff.on_exception(backoff.expo,Server5xxError,5,factor=2)
     def get_access_token(self):
-        if self.__access_token is not None and self.__expires > datetime.utcnow():
+        if self.dev_mode:
+            try:
+                self.__access_token = self.config['access_token']
+                self.__expires=strptime_to_utc(self.config['expires_in'])
+            except KeyError as err:
+                raise Exception("Unable to locate key in config") from err
+            if not self.__access_token or self.__expires < now():
+                raise Exception("Access Token in config is expired, unable to authenticate in dev mode")
```

Comment:
Use either `ex` or `e` as a convention for exception.

```suggestion
            except KeyError as ex:
                raise Exception("Unable to locate key in config") from ex
            if not self.__access_token or self.__expires < now():
                raise Exception("Access Token in config is expired, unable to authenticate in dev mode")
```